config/settings/base.py

```diff
@@ -87,13 +87,26 @@
     "dj_rest_auth.registration",
     "rest_framework_simplejwt",
     "django_rest_passwordreset",
+    "channels",
 ]
 
+ASGI_APPLICATION = "plant.config.settings.asgi.application"
```

Comment:
ASGI_APPLICATION and CHANNEL_LAYERS variables in the config/local.py have values duplicated from the config/base.py so they can be removed as local.py by default imports all of the content from base